### PR TITLE
Rating snackbar

### PR DESCRIPTION
**Pull request by surbhi84**

### CODE REVIEW

File: components/snackbar/index.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Surbhi Kukreti" />
+    <meta name="description" content="snackbar component" />
+
+    <link rel="stylesheet" href="/styleGuide/main.css" />
+
+    <link rel="stylesheet" href="/components/button/button.css" />
+    <link rel="stylesheet" href="./snackbar.css" />
```
Comment (anupkgurung) [inline components/snackbar/index.html:13]: why are you using individual css files ?why not import it to one use that instead.

**Review by anupkgurung — APPROVED** [review]
looks good.

**Review by rohanmathur91 — APPROVED** [review]
Looks good to me.

File: components/snackbar/index.html (hunk 1 above)
Comment (surbhi84) [reply components/snackbar/index.html:13]: I was doing it upto now so that i know what files i'hv used...i'll improvise it in future.

**Pull request merged**